src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Remote/IRemoteAutoInsertService.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.ExternalAccess.Razor;
+using Microsoft.CodeAnalysis.Razor.Protocol.AutoInsert;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.CodeAnalysis.Razor.Remote;
+
+internal interface IRemoteAutoInsertService : IDisposable
+{
+    IEnumerable<string> TriggerCharacters {  get; }
```

Comment:
This feels like a very odd thing to put on a remote service. I'm not even sure if properties are supported. It also doesn't make sense that the trigger characters can be different between the OOP process and devenv, so not sure why it's here.